test/browser/component.js

```diff
@@ -297,24 +312,24 @@ describe(`Simple Component instance`, function () {
 
     it(`renders its template`, function () {
       expect(el.children).to.have.lengthOf(0);
-      expect(el.shadowRoot.children[1].className).to.equal(`foo`);
+      expect(el.shadowRoot.children[el.shadowRoot.children.length - 1].className).to.equal(`foo`);
     });
 
     it(`applies the styles`, function () {
-      expect(el.shadowRoot.children[0].innerHTML).to.equal(`color: blue;`);
+      expect(getComponentStylesheetText(el)).to.equal(`:host { color: blue; }`);
     });
 
     context(`when applying override styles`, function () {
       it(`appends the overriding styles to the default styles`, async function () {
-        el.setAttribute(`style-override`, `background: red;`);
+        el.setAttribute(`style-override`, `:host { background: red; }`);
```

Comment:
Is this just so that tests will work cross-browser? We're not making a backwards-incompatible change to the `style-override` API right?

Reply:
Yes that's correct, we're not making any backwards-incompatible changes but the test code for browsers that support `adoptedStylesheets` will balk at `background-red` because is actually _invalid_ CSS, which causes `adoptedStylesheet` to just ignore the CSS altogether.